inc/azure_macro_utils/macro_utils.h

```diff
@@ -195,6 +195,12 @@ const char* MU_C3(MU_, enumIdentifier,_ToString)(enumIdentifier value)
 
 #define MU_ENUM_TO_STRING_2(enumIdentifier, value) MU_C3(MU_, enumIdentifier,_ToString)(value)
 
+#define MU_PRI_ENUM "s%s (%d)"
```

Comment:
>MU_ [](start = 8, length = 3)

Can we have PRI_MU_ENUM? Because all others start with PRI

Reply:
Alright, we shall have

---
In reply to: [335809967](https://github.com/Azure/azure-macro-utils-c/pull/34#discussion_r335809967) [](ancestors = 335809967)